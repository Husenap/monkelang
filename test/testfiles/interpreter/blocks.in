monke has string a is "global a"
monke has string b is "global b"
monke has string c is "global c"

monke do
  monke has string a is "outer a"
  monke has string b is "outer b"
  monke do
    monke has string a is "inner a"
    monke say "a: " + a
    monke say "b: " + b
    monke say "c: " + c
  monke done
  monke say "a: " + a
  monke say "b: " + b
  monke say "c: " + c
monke done
  
monke say "a: " + a
monke say "b: " + b
monke say "c: " + c

