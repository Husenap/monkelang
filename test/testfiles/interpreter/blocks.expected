a: inner a
b: outer b
c: global c
a: outer a
b: outer b
c: global c
a: global a
b: global b
c: global c
